package.json

```diff
@@ -70,20 +82,35 @@
   "devDependencies": {
     "@capacitor/cli": "3.6.0",
     "@commitlint/cli": "^17.0.3",
+    "@storybook/addon-actions": "^6.5.9",
+    "@storybook/addon-essentials": "^6.5.9",
+    "@storybook/addon-interactions": "^6.5.9",
+    "@storybook/addon-links": "^6.5.9",
+    "@storybook/builder-webpack5": "^6.5.9",
+    "@storybook/manager-webpack5": "^6.5.9",
+    "@storybook/node-logger": "^6.5.9",
+    "@storybook/preset-create-react-app": "^4.1.2",
+    "@storybook/react": "^6.5.9",
+    "@storybook/testing-library": "^0.0.13",
     "@typescript-eslint/eslint-plugin": "^5.30.4",
     "@typescript-eslint/parser": "^5.30.4",
     "@vidavidorra/commitlint-config": "^4.0.4",
+    "babel-plugin-named-exports-order": "^0.0.2",
     "danger": "^11.0.7",
     "eslint": "^8.19.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-config-standard": "^17.0.0",
     "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-n": "^15.2.4",
```

Comment:
Only this import was added manually, storybook threw an error since eslint-config-standard depended on it but it was never imported